### DISCUSSION TITLE
Fractional Scaling on hackintoshes

### DISCUSSION THREAD

**Discussion by BlueNecko** [kalkmann/Legion-5600H-Hackintosh#15] [General]
Well good day fellas, and you too kalkmann for being successful in making this efi. This will help many newbies. I am from a little latter group of hackintoshers and at the time when i got this laptop [ 15ACH6D1 ] there was litterally no support or hope was very grim. Even chances of nootred being possible were very slim. but nonthless i turned this into a hack as soon as nootred got public and i have been using this since then. 

well everything works fine really but i have just one simple issue that is fractional scaling, i.e small things get too blurry or pixelated. it has bugged me since then and has been a major turnoff for me compared to windows where things look clear as a glass. i wonder if you have encountered this or have found a way to fix this issue. i will attach some screenshots of the same bellow. 

![Screenshot 2023-11-15 at 6 51 59 PM](https://github.com/kalkmann/Legion-5600H-Hackintosh/assets/66507691/70f4452c-6f39-4113-bde2-eabacb6f5c70)
![Screenshot 2023-11-15 at 6 53 41 PM](https://github.com/kalkmann/Legion-5600H-Hackintosh/assets/66507691/4b7e4299-1e3b-42c8-a054-8f3378ac666f)

**Comment by kalkmann** [comment]
My laptop has a dedicated GPU (natively supported) and I haven't had any experience with AMD's integrated ones yet (using kext nootedred).
